JSQMessagesViewController/Factories/JSQMessagesToolbarButtonFactory.m

```diff
@@ -54,7 +72,7 @@ + (UIButton *)defaultSendButtonItem
     [sendButton setTitleColor:[[UIColor jsq_messageBubbleBlueColor] jsq_colorByDarkeningColorWithValue:0.1f] forState:UIControlStateHighlighted];
     [sendButton setTitleColor:[UIColor lightGrayColor] forState:UIControlStateDisabled];
 
-    sendButton.titleLabel.font = [UIFont boldSystemFontOfSize:17.0f];
+    sendButton.titleLabel.font = self.buttonFont != nil ? self.buttonFont : [UIFont boldSystemFontOfSize:17.0];
```

Comment:
same, once we add the `init` above, we don't need this `nil` check and can just assign `self.buttonFont` 😄